### ISSUE TITLE
error with e() function (since 3.10.1)

### ISSUE THREAD

**Issue by cor3000** [less/less.js#3420]
compile this less code containing using the e() function (using 3.10.1)

reproduced with node versions: 10.16 LTS, 12.7, 12.8.1

```
body {
    color: red;
}

e('anything to unquote');
```

result
```
SyntaxError: Quoted node returned by a function is not valid here in /home/robert/main.less on line 5, column 1:
4 
5 e('anything to unquote');
6 
```

the same less code compiles as expected in 3.9.0

maybe related to https://github.com/less/less.js/issues/3345

**Comment by matthew-dean** [comment]
@cor3000 What are you trying to achieve? It's true you can't dump a quoted value in the root of a stylesheet, although I'm not sure why it worked before.

**Comment by matthew-dean** [comment]
@cor3000 Oh no, you're right, I did change behavior of the `e()` function in a PR after 3.9.0. I hadn't yet added the changelog for it. -> https://github.com/less/less.js/commit/88e126bfc53932b8090fe506dd9ad154d6fff81a

**Comment by matthew-dean** [comment]
@cor3000 It looks like I need to change the behavior to say that, yes, a quoted value is not allowed in the root, but an escaped quoted value is fine.

**Comment by cor3000** [comment]
I'm trying to what the documentation says: output " proprietary syntax which Less doesn't recognize."

In our case our css stylesheets contain non standard root level text that's processed at runtime.

**Comment by matthew-dean** [comment]
@cor3000 I've set it as a high-priority bug, but not sure when I or someone else can get to it. Hopefully this week.

**Comment by cor3000** [comment]
Thanks, so far using 3.9.0 works for us.

**Comment by cor3000** [comment]
How would I output a string like this, at root level?

```
e('<% taglib prefix="c" uri="http://someurl"%>');
```

I know this is not css but something our legacy pre-prossessor understands during runtime to add session specific information e.g. for locale dependent styles.

**Comment by matthew-dean** [comment]
@cor3000 I just pushed a new version, should work.

**Comment by cor3000** [comment]
@matthew-dean thanks for the quick fix, works like a charm on 3.10.3
